source/import-font-and-images-assets/index.html.md.erb

```diff
@@ -18,7 +18,7 @@ For example if you're using [express.js](https://expressjs.com/), add the follow
 
 ```javascript
 const path = require('path');
-app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/dist/govuk/assets')))
+app.use('/assets/', express.static(path.join(__dirname, '/node_modules/govuk-frontend/dist/govuk/assets/')))
```

Comment:
I think these flashes got added when we edited, then edited, this line. If it causes a problem in code I can remove them, but assume they're harmless at the moment.